BHoM_UI/BHoM_UI.csproj

```diff
@@ -63,6 +67,9 @@
       <HintPath>..\..\BHoM_Engine\Build\Library_Engine.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="MongoDB.Bson, Version=2.4.4.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Why is this necessary? Referring the `Serialization_Engine.dll` is not enough?

Reply:
I think that's been added by @adecler's changes.